ecs-init/docker/dependencies_test.go

```diff
@@ -32,27 +34,19 @@ var netError = &url.Error{Err: &net.OpError{Op: "read", Net: "unix", Err: io.EOF
 var httpError = &docker.Error{Status: http.StatusInternalServerError, Message: "error"}
 
 func TestIsNetworkErrorReturnsTrue(t *testing.T) {
-	if !isNetworkError(netError) {
-		t.Errorf("Expected true when checking if network error")
-	}
+	assert.True(t, isNetworkError(netError), "Expect network error to be true if netError passed in")
```

Comment:
nit: The error message is a little bit confusing. Maybe "Expect isNetworkError to return true if network error passed in"?